roles/cloud_connector/templates/foreman_rh_cloud.toml.j2

```diff
@@ -3,6 +3,6 @@ protocol = "grpc"
 env = [
   "FORWARDER_USER={{ foreman_cloud_connector_user }}",
   "FORWARDER_PASSWORD={{ foreman_cloud_connector_password }}",
-  "FORWARDER_URL={{ foreman_cloud_connector_url }}",
+  "FORWARDER_URL={{ foreman_cloud_connector_url }}/api/v2/rh_cloud/cloud_request",
```

Comment:
@ShimShtein can you verify?

Reply:
Sounds like something the worker should figure out on its own to me?

Reply:
Maybe this fix is wrong, and `foreman_cloud_connector_url` should be set to `https://$(hostnamectl --static)/api/v2/rh_cloud/cloud_request` by whatever/whoever is calling the playbook?

Reply:
No, the `foreman_cloud_connector_url` is the URL to the "root" of the Foreman Installation, as it's also used in other places in the playbook to call the Foreman API.

Reply:
> Sounds like something the worker should figure out on its own to me?

What do you mean? The worker is a generic worker that knows how to forward to a URL.

Reply:
> > Sounds like something the worker should figure out on its own to me?
> 
> What do you mean? The worker is a generic worker that knows how to forward to a URL.

Oh yeah, right, it's not foreman specific. Yeah, then here is the right place